### PR TITLE
Add setAlwaysOnTop support

### PR DESCRIPTION
**Pull request by Happy-Ferret**
Will create the corresponding PR for go-astilectron as soon as this PR has been reviewed and merged.

asticode/go-astilectron#360

### CODE REVIEW

File: src/consts.js [hunk 1]
```diff
@@ -102,6 +103,7 @@ module.exports = {
         windowCmdWebContentsCloseDevTools: "window.cmd.web.contents.close.dev.tools",
         windowCmdWebContentsOpenDevTools: "window.cmd.web.contents.open.dev.tools",
         windowCmdWebContentsExecuteJavascript: "window.cmd.web.contents.execute.javascript",
+        windowEventAlwaysOnTopChanged: "window.event.always.on.top.changed",
```
Comment (asticode) [inline src/consts.js:106]: Could you rename the event to `window.event.always.on.top.set` as well as the const to `windowEventAlwaysOnTopSet` ?

**Review by asticode — CHANGES_REQUESTED** [review]

File: src/consts.js (hunk 1 above)
Comment (Happy-Ferret) [reply src/consts.js:106]: Ok, sure.
Although the way I implemented it was based on the official Electron docs.

Comment (Happy-Ferret) [reply src/consts.js:106]: @asticode 

Here's the documentation for that event.
Is there a reasoning for wanting to differ from the reference implementation?

https://www.electronjs.org/docs/latest/api/browser-window#event-always-on-top-changed

Comment (Happy-Ferret) [reply src/consts.js:106]: IMHO, `changed` is preferable to `set` for multiple reasons.

1. It's in the reference implementation (Electron).
2. It's less ubiquitous. `set` implies the existence of an `unset` but there is only one event.

Comment (asticode) [reply src/consts.js:106]: fair enough 👍

**Review by asticode — APPROVED** [review]

**Comment by asticode** [comment]
fyi I've created a `v0.52.0` tag

**Pull request merged**